state-chain/runtime-upgrade-utilities/src/helper_functions.rs

```diff
@@ -1,18 +1,12 @@
-use frame_support::storage::{storage_prefix, unhashed};
+use frame_support::{migration::move_storage_from_pallet, traits::PalletInfoAccess};
 
-/// Not to be confused with [move_prefix]. This function move the storage identified by the given
-/// pallet and storage names.
-pub fn move_storage(
-	old_pallet_name: &[u8],
-	old_storage_name: &[u8],
-	new_pallet_name: &[u8],
-	new_storage_name: &[u8],
-) {
```

Comment:
Maybe we should not delete this utility for when we have to move and rename storages

Reply:
I deleted it because it didn't work 🙈